server/config/options.go

```diff
@@ -78,7 +78,7 @@ type TagsConfig struct {
 
 var DefaultConfig = Config{
 	Log: log.LogConfig{
-		Level: "trace",
+		Level: "info",
```

Comment:
Is the default change on purpose or maybe leftover?

Reply:
So do we need `trace` by default? I have no concern about changing it back. The clusters must be already setting it to info so this is just for local?

Reply:
Right, the clusters are using a production configuration, we kept the default configuration for the local run of a single developer.

Reply:
let me keep it to info for now as we are doing a lot of perf testing, later if needed we can change it to trace again.